# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/nikolaus/spice/linuxconf.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./conf/linux.org	Fri Jun 25 07:24:00 1999
+++ ./conf/linux	Tue May 27 10:18:22 2003
@@ -14,7 +14,7 @@
 
 CC_OPT	        = -O2 -s
 
-SPICE_DIR       = /usr/local/spice
+SPICE_DIR       = /usr/spice
 SPICE_LIB_DIR   = $(SPICE_DIR)/lib
 SPICE_EXEC_DIR  = $(SPICE_DIR)/bin
 
